timrlink.net.Core/Service/TaskService.cs

```diff
@@ -83,6 +84,12 @@ public async Task UpdateTask(API.Task task)
             await timrSync.UpdateTaskAsync(new API.UpdateTaskRequest(task)).ConfigureAwait(false);
         }
 
+        public async Task DeleteTask(API.Task task)
+        {
+            logger.LogInformation($"Deleting Task(Name={task.name}, ExternalId={task.externalId})");
+            await timrSync.DeleteTaskAsync(new API.DeleteTaskRequest(task.externalId)).ConfigureAwait(false);
```

Comment:
There is also a `DeleteTaskByUUID()` which I would prefer over the externalId variant. That one would always work, but the current one would fail for Tasks without externalId.

We could additionally add a `DeleteTaskByExternalId(string externalId)` function if we like.